src/DevCenter/DevCenter.AutoRest/custom/New-AzDevCenterAdminPool.ps1

```diff
@@ -111,60 +111,66 @@ function New-AzDevCenterAdminPool {
         # To construct, see NOTES section for INPUTOBJECT properties and create a hash table.
         ${InputObject},
     
-        [Parameter(ParameterSetName = 'CreateExpanded', Mandatory)]
-        [Parameter(ParameterSetName = 'CreateViaIdentityExpanded', Mandatory)]
+        [Parameter(Mandatory)]
```

Comment:
By expanding the parametersets where the Location must be Mandatory, there will be breaking changes. Do you know the context?